hack/collector-default

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
+# Description                                                                      #
+# -------------------------------------------------------------------------------- #
+# collctor-default will collect host information as package for support-bundle-kit #
+# e.g., hostname, syslog, dmesg etc                                                #
+# -------------------------------------------------------------------------------- #
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+# -------------------------------------------------------------------------------- #
+# host information                                                                 #
+# -------------------------------------------------------------------------------- #
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
+
+# Boot Configuration
+cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+
+# CPU, Memory and Process usage status
+top -b -n 1 > cpu_memory_process_status
+
+# -------------------------------------------------------------------------------- #
+# collect logs                                                                     #
+# -------------------------------------------------------------------------------- #
+cd ${BUNDLE_DIR}
+mkdir -p logs
+cd logs
+
+dmesg &> dmesg.log
+
+# syslog
+tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
+
+#kernel log
+chroot ${HOST_PATH} /usr/bin/journalctl -r -k > kernel.log
+
+# kubelet log
+# systemd is present, kubelet log writes to journald
+chroot ${HOST_PATH} /usr/bin/journalctl -r -u kubelet > kubelet_journald.log
+# systemd is not present, kubelet log are written to a `.log` file which is kept in `/var/log` or a redirected target folder
+find ${HOST_PATH} -iname kubelet.log | xargs -I % cp % .
```

Comment:
Using `find`'s cost is too high.

I think we can have a matrix like
{
   "k3s": "..log",
   "rke2": "..log",
   "....",
}
and just check the these paths.